package/python-kiwi-spec-template

```diff
@@ -746,6 +746,16 @@ fi
 %endif
 %endif
 
+%post -n dracut-kiwi-oem-repart
+#============================================================
+# create /etc/dracut.conf.d/40-kiwi-resize.conf
+mkdir -p /etc/dracut.conf.d
+echo 'add_dracutmodules+=" kiwi-repart "' > /etc/dracut.conf.d/40-kiwi-resize.conf
+echo 'install_items+=" /config.partids "' >>/etc/dracut.conf.d/40-kiwi-resize.conf
+if [ -e "/.profile" ];then
```

Comment:
I understand this is only likely to be there if someone installs this package as part of a KIWI build. So this looks safe good to me.